Exercise_C.js

```diff
@@ -13,6 +13,18 @@ If the number is divisible by 3 and 5 it should log Hong Kong.
 See README.md if you need help running your code.
 */
 
+
+
 for (let i = 1; i <= 30; i++) {
-  // CODE HERE
+  if (i%3 == 0 && i%5!=0) {
+    console.log("Hong");
+  }
+  else if (i%5 == 0 && i%3!=0) {
+    console.log("Kong");
+  }
+  else if (i%5 == 0 && i%3==0){
+    console.log("Hong Kong");
+}
+  else 
+    console.log(i);
 }

```

Comment:
Decent implementation!
A way to remove the extra condition you added to line 19
would be to swap the condition and code block starting on 25 with line 19.
Consider the order of your if and else statements